diff --git a/node_modules/graphql-upload/index.js b/node_modules/graphql-upload/index.js
new file mode 100644
index 0000000..cef28ab
--- /dev/null
+++ b/node_modules/graphql-upload/index.js
@@ -0,0 +1,6 @@
+module.exports = {
+  Upload: require('./Upload.js'),
+  GraphQLUpload: require('./GraphQLUpload.js'),
+  graphqlUploadKoa: require('./graphqlUploadKoa.js'),
+  graphqlUploadExpress: require('./graphqlUploadExpress.js'),
+};
